src/tests/eet/eet_suite.c

```diff
@@ -8,6 +8,14 @@
 #include "../efl_check.h"
 #include <Eet.h>
 
+#ifdef _MSC_VER
+/* On windows, applink.c needs to be included in the executable; otherwise
+ * all the tests which are using file pointer will fail in runtime, throwing
+ * `No OPENSSL_Applink` messages, i.e: eet_test_identity.
+ * */
+# include <openssl/applink.c>
+#endif
```

Comment:
Does it need to be specifically at `exe`? Could it be at a `dll`? I mean could we put it at `avil` guarded with `HAVE_OPENSSL` (which is defined at `meson.build`)

Reply:
> Does it need to be specifically at `exe`?

Yes, [as stated in OpenSSL FAQ](https://www.openssl.org/docs/faq.html#PROG3).

> I mean could we put it at `evil` guarded with `HAVE_OPENSSL` (which is defined at `meson.build`)

An alternative could be make a `INCLUDE_OPENSSL_APPLINK` symbol that is only true when compiling the executable, but to be honest if we would do this, I prefer to just ask to include `<openssl/applink.c>`.